apollo-codegen.ts

```diff
@@ -11,6 +11,8 @@ const config: CodegenConfig = {
       plugins: ['typescript', 'typescript-operations', 'typescript-react-apollo'],
       config: {
         withHooks: true,
+        // This avoid all generated schemas being wrapped in Maybe https://the-guild.dev/graphql/codegen/plugins/typescript/typescript#maybevalue-string-default-value-t--null
```

Comment:
is it safe though?

Reply:
Yes, all values can still be undefined and will require that check. This prevent redundant checking for both null and undefined